pkg/lib/cockpit-components-password.jsx

```diff
@@ -62,30 +66,38 @@ export const PasswordFormFields = ({
     return (
         <>
             <FormGroup label={password_label}
+                       labelIcon={password_label_info &&
+                           <Popover bodyContent={password_label_info}>
+                               <button onClick={e => e.preventDefault()}
+                                       className="pf-c-form__group-label-help">
+                                   <HelpIcon noVerticalAlign />
+                               </button>
+                           </Popover>
+                       }
```

Comment:
I cannot reproduce this functionality. Does it work? How do I make the popover to happen?

Reply:
sure it works, choose fedora, click on Run unattended installation and you will see next to the root password a ? popover button.